src/recovery.rs

```diff
@@ -51,6 +52,8 @@ const INITIAL_RTT: Duration = Duration::from_millis(500);
 
 const PERSISTENT_CONGESTION_THRESHOLD: u32 = 3;
 
+const RTT_WINDOW: Duration = Duration::from_secs(10);
```

Comment:
This default is too small. Let's start with 300sec (I think that's what Chrome uses)